models/base/pageviews_proc.sql

```diff
@@ -31,22 +28,13 @@ max(if(params.key = 'campaign', params.value.string_value, null)) utm_campaign,
 max(if(params.key = 'page_referrer', params.value.string_value, null)) utm_referrer,
 max(ecommerce.transaction_id) ecommerce_transaction_id,
 max(ecommerce.purchase_revenue) ecommerce_purchase_revenue
-FROM 
-
--- Check if _intraday or daily events tables should be used
-{% if intraday_tables|length > 0 %}
-	`{{ target.project }}.{{ target.schema }}.events_intraday_*`,
-{% else %}
-	`{{ target.project }}.{{ target.schema }}.events_*`,
-{% endif %}
-
+FROM
+`{{ target.project }}.{{ target.schema }}.events_*`,
 UNNEST(event_params) AS params
 WHERE event_name = 'page_view'
 
 -- Refresh only recent session data to limit query costs, unless running with --full-refresh
-{% if is_incremental() %}
```

Comment:
@jhnvdw the {% if is_incremental() % } statement here allows us to reduce BQ query costs, by only incrementally building the model for days that are within the lookback period